src/components/toolbars/content-toolbar.js

```diff
@@ -153,44 +142,53 @@ const ContentToolbar = (props) => {
         [selectedElements, noCreationInProgress]
     );
 
-    const items = useMemo(
-        () =>
+    const items = useMemo(() => {
+        const items = [];
+
+        if (
             selectedElements.length &&
             (allowsDelete || allowsMove || allowsDownloadCases)
-                ? [
-                      {
-                          tooltipTextId: 'delete',
-                          callback: () => {
-                              handleOpenDialog(DialogsId.DELETE);
-                          },
-                          icon: <DeleteIcon fontSize="small" />,
-                          disabled: !selectedElements.length || !allowsDelete,
-                      },
-                      {
-                          tooltipTextId: 'move',
-                          callback: () => {
-                              handleOpenDialog(DialogsId.MOVE);
-                          },
-                          icon: <DriveFileMoveIcon fontSize="small" />,
-                          disabled: !selectedElements.length || !allowsMove,
-                      },
-                      {
-                          tooltipTextId: 'download.button',
-                          callback: () => handleDownloadCases(selectedElements),
-                          icon: <FileDownload fontSize="small" />,
-                          disabled:
-                              !selectedElements.length || !allowsDownloadCases,
-                      },
-                  ]
-                : [],
-        [
-            allowsDelete,
-            allowsDownloadCases,
-            allowsMove,
-            handleDownloadCases,
-            selectedElements,
-        ]
-    );
+        ) {
+            items.push({
+                tooltipTextId: 'delete',
+                callback: () => {
+                    handleOpenDialog(DialogsId.DELETE);
+                },
+                icon: <DeleteIcon fontSize="small" />,
+                disabled: !selectedElements.length || !allowsDelete,
+            });
+
+            items.push({
+                tooltipTextId: 'move',
+                callback: () => {
+                    handleOpenDialog(DialogsId.MOVE);
+                },
+                icon: <DriveFileMoveIcon fontSize="small" />,
+                disabled: !selectedElements.length || !allowsMove,
+            });
+
+            items.push({
+                tooltipTextId: 'download.button',
+                callback: () => handleDownloadCases(selectedElements),
+                icon: <FileDownload fontSize="small" />,
+                disabled: !selectedElements.length || !allowsDownloadCases,
+            });
+        }
+
+        items.push({
+            tooltipTextId: 'StashedElements',
+            callback: () => handleOpenDialog(DialogsId.STASHED_ELEMENTS),
+            icon: <RestoreFromTrash fontSize="small" />,
+        });
+
+        return items;
```

Comment:
Why successive "pushs" instead of single allocation like [a, b, ...]

Reply:
i replace them with single push for all items